### ISSUE TITLE
Events: Past events need to be re-added to keystone

### ISSUE THREAD

**Issue by kanarinka** [engagementgamelab/boston-civic-media#76]
Just wondering where the past events are?

**Comment by jpailthorpe** [comment]
these need to be re-added in keystone because we updated the models with the new designs. I can do this today if it is high priority.

**Comment by jpailthorpe** [comment]
https://drive.google.com/open?id=0B83JjjyA9r52QVdSRXNJRnFEbmM

more events in timeline 
https://cdn.knightlab.com/libs/timeline3/latest/embed/index.html?source=1nPAwCDRMHNc5fuhobxhqs-4zZ4lrM3glZemB_FBj7mc&font=Default&lang=en&initial_zoom=.5&height=650

**Comment by jpailthorpe** [comment]
I placed most of the big name conferences back into keystone. The images are def. off since most of them just had the generic logo. I can put that logo back in as the image if you'd like i just need to format it into the proper format. 

If there are other events you would like to put in, they can be added in keystone. I linked the folder above. I suggest making a new issue with specific info that needs to be added if there is something you want addressing. That said, anyone should be able to edit in keystone on the production server bostoncivic.media/keystone)
